site/docs/multi-engine-support.md

```diff
@@ -67,7 +67,7 @@ Each engine version undergoes the following lifecycle stages:
 | 3.0        | End of Life        | 0.9.0                   | 1.0.0                  | [iceberg-spark-runtime-3.0_2.12](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.0_2.12/1.0.0/iceberg-spark-runtime-3.0_2.12-1.0.0.jar) |
 | 3.1        | End of Life        | 0.12.0                  | 1.3.1                  | [iceberg-spark-runtime-3.1_2.12](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.1_2.12/1.3.1/iceberg-spark-runtime-3.1_2.12-1.3.1.jar) [1] |
 | 3.2        | End of Life        | 0.13.0                  | 1.4.3                  | [iceberg-spark-runtime-3.2_2.12](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.2_2.12/1.4.3/iceberg-spark-runtime-3.2_2.12-1.4.3.jar) |
-| 3.3        | End of Life        | 0.14.0                  | 1.8.0                  | [iceberg-spark-runtime-3.3_2.12](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.3_2.12/1.8.0/iceberg-spark-runtime-3.3_2.12-1.8.0.jar) |
+| 3.3        | Deprecated         | 0.14.0                  | {{ icebergVersion}}    | [iceberg-spark-runtime-3.3_2.12](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.3_2.12/1.8.0/iceberg-spark-runtime-3.3_2.12-1.8.0.jar) |
```

Comment:
Since this PR is for main branch, not the 1.7.x branch. 
I don't think we need to revert the EOL as it is removed for 1.8.0. 

May be raise a PR for 1.7.x branch for this change.

Reply:
Spark 3.3 is removed on the main branch and 1.9, and lives as long as 1.8 release. BTW, there's no versioned doc for this page.

Reply:
I see. I agree with this change. 

Btw there is 1.8.1 release in progress. We might have to update accordingly.

Reply:
We are using `{{ icebergVersion}}` here, so it will update itself.

Reply:
But the jar version is hardcoded to `iceberg-spark-runtime-3.3_2.12-1.8.0.jar`, need to update it to use {{ icebergVersion}}

Reply:
Nice catch! Updated now. Thanks!